sites/nau/src/frontend/scss/extras/components/_footer.scss

```diff
@@ -54,20 +54,42 @@
     &>a {
       $nesteditem-base-selector: ".nested-item";
 
+      #{$nesteditem-base-selector}__content {
+        font-size: 1rem;
+      }
+
       #{$nesteditem-base-selector}__items {
         li {
           padding: 0.1rem 0.1rem 0.1rem 0rem;
+
+          a[href*="//"]:not([href*="nau.edu.pt"]),
+          a[target="_blank"] {
```

Comment:
Having nau.edu.pt hardcoded seams not a good solution, please remove it.
```css
a[href*="//"]:not([href*="nau.edu.pt"]),
```

Having a special case for target _blank cases is ok. I tried adding for all site but isn't practical, because of the social sharing stuff. 
```css
a[target="_blank"]
```

But having the same code repeated N times isn't a good solution.
Can you create a scss function/mixin and add it to the places that need to have that special arrow?